### DISCUSSION TITLE
How to make initial claim blocks per groups ?

### DISCUSSION THREAD

**Discussion by Mario359** [GriefPrevention/GriefPrevention#731] [Q&A] [answered]
Hi, I'm trying to set a default number of claim blocks for each group on my server.

So I used _griefprevention.initial-claim-blocks_ from: https://github.com/MinecraftPortCentral/GriefPrevention/wiki/Advanced-Options
And I typed the command: _/lp group c meta set griefprevention.initial-claim-blocks 8192_

But it didn't work, players still have 4096 blocks (in the plugin configuration) although they are in this group

I also tried the same command with contexts but it still didn't work
![image](https://user-images.githubusercontent.com/20285750/75266666-f7d73600-57f2-11ea-8498-49c6c64c490b.png)
![image](https://user-images.githubusercontent.com/20285750/75266807-2d7c1f00-57f3-11ea-9498-73682c601122.png)

Did I miss something?

**Comment by bigpresh** [comment]
I think you have the wrong issue tracker - this is for the original Spigot/Paper version, and you're referring to a feature from the Sponge version at https://github.com/MinecraftPortCentral/GriefPrevention

**Comment by Mario359** [comment]
Oh, so there is no way to do the same with Paper ?

Note: I just found GPAccruals on https://github.com/TechFortress/GriefPrevention/wiki/Addons can I use it for the lastest versions of Paper & GriefPrevention ? (I'll try it when I'm on my PC)

**Comment by RoboMWM** [comment, marked as answer]
> Note: I just found GPAccruals on https://github.com/TechFortress/GriefPrevention/wiki/Addons can I use it for the lastest versions of Paper & GriefPrevention 

Yes.

And yea, you linked the wrong wiki. I'm not sure how more and more people seem to be finding that plugin, perhaps it's because I mention "GitHub" and theirs happens to be high up in the results, so that isn't very helpful...

**Comment by Mario359** [comment]
So, I tried to use the plugin GPAccruals, adding new permissions, but it still not working
In the config of this plugin I created:
```accruals:
  increment:
    one: 1
  maxcap:
    c: 8192
    b: 12288
    a: 16384
    s: 24576
    z: 32768
```
With _/lp tree_, permissions are here
![image](https://user-images.githubusercontent.com/20285750/75489980-91e5dc80-59b3-11ea-8012-feb6d278a55d.png)
Then I added the permission griefprevention.accruals.maxcap.c for group c
Players still get 4096 blocks and not 8192

**Comment by RoboMWM** [comment]
We don't manage the addon. I'll add the bitbucket link to that.

**Comment by Mario359** [comment]
Okay, I'll go check over there, thanks for your answer
